airgun/entities/file.py

```diff
@@ -0,0 +1,51 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.utils import retry_navigation
+from airgun.views.file import FileDetailsView, FilesView
+
+
+class FilesEntity(BaseEntity):
```

Comment:
Can you add doc string for classes and methods written in the entitites>file.py

Reply:
It's pretty much self explanatory, imo